src/main/java/com/rybka/command/ConvertCommand.java

```diff
@@ -0,0 +1,16 @@
+package com.rybka.command;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class ConvertCommand implements Command {
+    private final ConvertTask convertTask;
+
+    @Override
+    public void execute() {
+        var convertedResult = convertTask.convert();
```

Comment:
Why do you need ConvertTask then? 

put all logic and dependencies in the ConverCommand. 

In result you will have

ConvertCommand
ExportCommand
BlaBlaCommand

Reply:
fixed